src/Sylius/Behat/Context/Api/Shop/LocaleContext.php

```diff
@@ -85,9 +90,23 @@ public function theLocaleWithCodeShouldNotBeAvailable(string $name, string $code
      */
     public function iShouldHaveWithCode(string $name, string $code): void
     {
-        $response = $this->client->getLastResponse();
+        $response = $this->localesClient->getLastResponse();
 
         Assert::true($this->responseChecker->hasValue($response, 'name', $name));
         Assert::true($this->responseChecker->hasValue($response, 'code', $code));
     }
+
+    /**
+     * @Then I should shop using the :localeCode locale
+     * @Then I should still shop using the :localeCode locale
```

Comment:
```suggestion
     * @Then I should( still) shop using the :localeCode locale
```
and line 100 will be redundant :)